src/CONST.ts

```diff
@@ -2606,7 +2606,9 @@ const CONST = {
             NAVIGATE: 'NAVIGATE',
         },
     },
-
+    INDENTS: '    ',
+    PARENT_CHILD_SEPARATOR: ': ',
+    CATEGORY_LIST_THRESHOLD: 8,
```

Comment:
I wonder if we can get more specific with the name here. Maybe `CATEGORY_LIST_SEARCH_BAR_THRESHOLD`?

Reply:
Basically, it is not a search related threshold. It is a threshold for a list itself, because it is being taken into account when previewing entire list.